test/plugins/features/movies/index.test.js

```diff
@@ -84,11 +128,61 @@ describe('movies integration', () => {
           'id',
           'title',
           'release_year',
+          'locations',
           'object'
         ]);
       });
     });
 
   });
 
+  describe('addLocation', () => {
+
+    it('location to movie', () => {
+      return Bluebird.all([
+        new Movie({ title: MOVIE_LIST[2] }).fetch(),
+        new Location({ name: LOCATION_LIST[0] }).fetch()
+      ])
+      .spread((movie, location) => {
+        return Movies.inject({
+          url: `/movies/${movie.id}/locations`,
+          method: 'POST',
+          payload: { id: location.id }
+        });
+      })
+      .then((response) => {
+        expect(response.statusCode).to.eql(200);
+      });
+    });
+
+    it('adds a location to a non-existent movie', () => {
```

Comment:
`errs when passed a non-existent movie`

actually, i think for both of these `404` errors, you should add them as controller tests instead of integration tests since this is logic that is in the controller and integration tests tend to take a bit longer (since it goes through the whole request lifecycle) so we try to minimize the amount we have